api/graphql/tests/test_users.py

```diff
@@ -0,0 +1,293 @@
+import json
+
+import snapshottest
+from assertpy import assert_that
+from django.contrib.auth import get_user_model
+from graphene_django.utils import GraphQLTestCase
+
+from permissions.models import (
+    GeneralRole,
+    GeneralRoleChoice,
+    ServiceSectorRole,
+    ServiceSectorRoleChoice,
+    UnitRole,
+    UnitRoleChoice,
+)
+from spaces.tests.factories import ServiceSectorFactory, UnitFactory, UnitGroupFactory
+from users.models import ReservationNotification, User
+
+
+class UserQueryTestCaseBase(GraphQLTestCase, snapshottest.TestCase):
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
Since both Query and Update tests share the same setup, I renamed this base class to `UserTestCaseBase` and made both test classes to inherit this. That way I got rid of some extra code and don't have to repeat the same setup ☺️

Reply:
oh, didn't notice that :+1: 
![](https://media.giphy.com/media/10RgsuetO4uDkY/giphy.gif)